src/sentry/migrations/0326_backfill_app_id_str.py

```diff
@@ -0,0 +1,45 @@
+# Generated by Django 2.2.28 on 2022-10-20 13:50
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def backfill_app_id_str(apps, schema_editor):
+    AppConnectBuild = apps.get_model("sentry", "AppConnectBuild")
+
+    for appconnect_build in RangeQuerySetWrapperWithProgressBar(AppConnectBuild.objects.all()):
+        appconnect_build.app_id_str = str(appconnect_build.app_id)
+        appconnect_build.save()
```

Comment:
Nit: You can use `.save(update_fields=["app_id_str"])` to just update the specific field.

Reply:
Does that do the int/str conversion automatically?